src/ReleaseHistory.md

```diff
@@ -1,12 +1,14 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
 ## UNRELEASED
+
 * BUGFIX: Eliminate dispose of stream and `StreamWriter` arguments passed to `SarifLog.Save` helpers. This would result in `ObjectDisposedException` being raised on attempt to access streams after save.
+* BREAKING: `Id` property of `Location` changed from `int`(32bit) to `BigInteger`(unlimited) to fix `Newtonsoft.Json.JsonReaderException: JSON integer XXXXX is too large or small for an Int32.` [#2463](https://github.com/microsoft/sarif-sdk/pull/2463)
```

Comment:
removed the very first part "error ERR999.UnhandledEngineException" which only apply to the SDK validate tool.

the rest error message apply to both SDK validate tool and directly using:
`JsonConvert.DeserializeObject<SarifLog>(content)`